### PR TITLE
Story/io 205/planning list group structure

### PR DESCRIPTION
**Pull request by EngbergJonas**

### CODE REVIEW

File: src/utils/interceptors.ts [hunk 1]
```diff
@@ -60,7 +85,11 @@ const handleError = (error: AxiosError): Promise<IError> => {
     }),
   );
 
-  store.dispatch(clearLoading());
+  console.log('Error intercepted: ', error);
```
Comment (kgunga) [inline src/utils/interceptors.ts:88]: Remove this debugging log

**Review by kgunga — CHANGES_REQUESTED** [review]

File: src/components/Planning/PlanningTable/PlanningRow/ProjectRow/ProjectRow.tsx [hunk 2]
```diff
@@ -47,48 +47,72 @@ const PlanningGroupsTableRow: FC<IPlanningGroupsTableRowProps> = ({ project }) =
...
               {project.name}
-            </button>
+            </Link>
           </div>
           {/* Right side (category & budget) */}
           <div className="project-right-icons-container">
             <div>
-              {project.category && <CustomTag text={project.category.value} weight={'light'} />}
+              {project.category && (
+                <>
```
[the remark this answers is not in the document]
Comment (EngbergJonas) [reply src/components/Planning/PlanningTable/PlanningRow/ProjectRow/ProjectRow.tsx]: fixed

File: src/utils/interceptors.ts (hunk 1 above)
Comment (EngbergJonas) [reply src/utils/interceptors.ts]: fixed

**Review by kgunga — APPROVED** [review]
👍

**Pull request merged**